pyoverkiz/enums/state.py

```diff
@@ -196,6 +203,7 @@ class OverkizState(str, Enum):
     RTDS_CONTROLLER_ORIGINATOR = "rtds:ControllerOriginatorState"
     RTDS_CONTROLLER_SENSING = "rtds:ControllerSensingState"
     RTDS_CONTROLLER_SIREN = "rtds:ControllerSirenState"
+    SLEEPING_MODE = "sleepingMode"
```

Comment:
This is also not in the right place?

Reply:
Sorry, you are right. I misread your comment: https://github.com/iMicknl/python-overkiz-api/pull/492/files

Reply:
You can use command.py OverkizCommandParam.

Reply:
Done